API/src/app/app.module.ts

```diff
@@ -0,0 +1,24 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { DatabaseType, getMetadataArgsStorage } from 'typeorm';
+import { CategoryModule } from './Category';
+import { DataPointModule } from './DataPoint/dataPoint.module';
+import { DSLModule } from './DataSetLayer/DSL.module';
+const mongodbConnection: DatabaseType = 'mongodb';
+@Module({
+    imports: [
+        DSLModule,
+        DataPointModule,
+        CategoryModule,
+        TypeOrmModule.forRoot({
+            type: mongodbConnection as any,
+            host: process.env.DB_HOST,
+            database: process.env.DB_DATABASE,
```

Comment:
Same question with one below that what happens if one of those is null? 
And I was wondering that we should put all retrieving env variables in 1 module/file or we use it directly in different files?

I think it is easier for us to change if we put them all in one file. Like we want to change variable name in .env file? Also, we may repeat running process.env many times when we scatter them in many files. 
On other hand, use them directly is easier and faster to develop.

Reply:
I dont think this is a big problem since we can refactor our code quickly later.